src/Boost.sol

```diff
@@ -43,11 +40,13 @@ contract Boost is IBoost, EIP712, Ownable, ERC721URIStorage {
     mapping(address => uint256) public tokenFeeBalances;
 
     // Constant eth protocol fee (in wei) that must be paid by all boost creators
-    uint256 public ethFee;
+    uint128 public ethFee;
+
+    // The id of the next boost to be minted
+    uint64 public nextBoostId;
 
-    // The fraction of the total boost deposit that is taken as a protocol fee
-    // represented as an integer denominator (100/x)%
-    uint256 public tokenFee;
+    // Per-myriad (parts per ten-thousand) of the total boost deposit that is taken as a protocol fee
```

Comment:
could be explicit in the comment its a number in range `[0,10000]`

Reply:
6f49dfac68ba775197d6cdb622183fee1f409c45